store/src/app/products/components/button/button.component.scss

```diff
@@ -0,0 +1,3 @@
+.product-card__button {
+  margin-top: 31px;
```

Comment:
Not really good solution. Button is like really common component, so it could be placed in a lot of different palces. So it shouldn`t postition himself.

Reply:
done